src/containers/Network/BarChartVersion.tsx

```diff
@@ -108,7 +110,14 @@ const BarChartVersion = (props: Props) => {
               dy={80}
             />
           </YAxis>
-          <Bar dataKey="value" barSize={30} fill={PURPLE} radius={[4, 4, 0, 0]}>
+          <Bar
+            dataKey="value"
+            barSize={30}
+            fill={PURPLE}
+            radius={[4, 4, 0, 0]}
+            onMouseOver={() => setShowTooltips(true)}
```

Comment:
What are the performance implications of this change? Is it really slow/does it make other things really slow?

Reply:
It shouldn't be slow these events don't continually fire.